lte/gateway/python/magma/pipelined/app/uplink_bridge.py

```diff
@@ -256,6 +256,14 @@ def _set_sgi_ipv4_and_v6_ingress_flows(self):
             actions = "output:LOCAL"
             self._install_flow(flows.MEDIUM_PRIORITY + 1, match, actions)
 
+        # forward the node solicite msg to local machine
+        addr = SOLICITED_NODE_MULTICAST
```

Comment:
Could this be inlined? 'addr' is not used anywhere else.

Reply:
@vktng fixed